### PR TITLE
Add synthesize videos.

### PR DESCRIPTION
**Pull request by tony2016uestc**
Synthesize videos based on RGB frames.

### CODE REVIEW

File: tools/data/build_videos.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import cv2
+
+
+def synthesize_video(frame_dir_item, dev_id=0):
+    """Generate videos using opencv-python.
+
+    Args:
+        frame_dir_item (list): Rawframe item containing raw frame directory
+            full path, rawframe directory (short) path, rawframe directory id.
+        dev_id (int): Device id.
```
Comment (dreamerlin) [inline tools/data/build_videos.py:16]: Is `dev_id` necessary ?

File: tools/data/build_videos.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+from multiprocessing import Pool
+
+
+def synthesize_video(frame_dir_item, dev_id=0):
+    """Generate videos using opencv-python.
+
+    Args:
+        frame_dir_item (list): Rawframe item containing raw frame directory
+            full path, rawframe directory (short) path, rawframe directory id.
+        dev_id (int): Device id.
```
Comment (dreamerlin) [inline tools/data/build_videos.py:15]: Is `dev_id` necessary?

File: tools/data/build_videos.py [hunk 3]
```diff
@@ -0,0 +1,126 @@
+import argparse
+import glob
+import os
+import os.path as osp
+import sys
+from multiprocessing import Pool
+
+
+def synthesize_video(frame_dir_item):
+    """Generate videos using opencv-python.
```
Comment (innerlee) [inline tools/data/build_videos.py:10]: Encode frames to video using ffmpeg

File: tools/data/build_videos.py [hunk 4]
```diff
@@ -0,0 +1,126 @@
+import argparse
+import glob
+import os
+import os.path as osp
+import sys
+from multiprocessing import Pool
+
+
+def synthesize_video(frame_dir_item):
```
Comment (innerlee) [inline tools/data/build_videos.py:9]: synthesize_video -> encode_video

File: tools/data/sthv1/encode_videos.sh [hunk 5]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+cd ../
+python build_videos.py ../../data/sthv1/rawframes/ ../../data/sthv1/videos/ --fps 12 --level 2 --start-idx 1 --filename-tmpl '%05d'
+echo "Genearte videos"
```
Comment (innerlee) [inline tools/data/sthv1/encode_videos.sh:5]: and here.

generate means create something completely new. here we are encoding images to videos---a kind of format transformation

**Pull request merged**